spec/integration/job_queues_spec.rb

```diff
@@ -14,6 +14,7 @@ module Backend
       let(:queue) { 'priority' }
 
       before do
+        Resque::Failure.clear
```

Comment:
This is something that should be done between any two tests no? So, a global hook ?

Reply:
If I am not wrong, default hook of `before` is `:each`, to between any two tests

Reply:
I meant globally, for the whole test suite. This is something that can happen and leak across different files, right?

Reply:
oh, you mean in `RSpec.configure` so it will be cleared between any two tests of the whole test suite.

I will put  there